src/vue/backend/src/store/index.js

```diff
@@ -5,6 +5,20 @@ Vue.use(Vuex)
 
 export default new Vuex.Store({
     state: {
+        relatedLinks: [
```

Comment:
The links are going to be different based on that page they are on, so having a `relatedLinks` stand alone will not be reusable